apps/ftu/index.html

```diff
@@ -19,7 +19,6 @@
   <link rel="stylesheet" href="style/style.css">
   <link rel="stylesheet" href="style/status.css">
   <link rel="stylesheet" href="style/overlay.css">
-  <link rel="stylesheet" href="style/fb_extensions.css">
   <link rel="stylesheet" href="style/tutorial.css">
```

Comment:
If we aren't using the fb_extensions.css file anymore, we should remove it. There's also fb_* ids in style.css we can remove from the rule   
#import_contacts #fb_import, #import_contacts #fb_after_import2 {
  margin-top: 1rem;
}